cmd/lncli/walletrpc_active.go

```diff
@@ -1193,6 +1195,191 @@ func listAddresses(ctx *cli.Context) error {
 	return nil
 }
 
+var signMessageWithAddrCommand = cli.Command{
+	Name:  "signmessage",
+	Usage: "Sign a message with the private key of the provided address",
+	Description: `
+	Sign a message with the private key of the specified address, and 
+	return the signature. Signing is solely done in the ECDSA compact 
+	signature format. This is also done when signing with a P2TR address
+	meaning that the private key of the P2TR address (internal key) is used
+	to sign the provided message with the ECDSA format. Only addresses are 
+	accepted which are owned by the internal lnd wallet.
+	`,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "msg",
+			Usage: "the message to sign for",
+		},
+		cli.StringFlag{
+			Name: "address",
+			Usage: "specify the address which private key " +
+				"will be used to sign the message",
+		},
+	},
+	Action: actionDecorator(signMessageWithAddr),
+}
+
+func signMessageWithAddr(ctx *cli.Context) error {
+	ctxc := getContext()
+
+	// Display the command's help message if we do not have the expected
+	// number of arguments/flags.
+	if ctx.NArg() > 2 || ctx.NumFlags() > 2 {
+		return cli.ShowCommandHelp(ctx, "signmessagewithaddr")
+	}
+
+	walletClient, cleanUp := getWalletClient(ctx)
+	defer cleanUp()
+
+	var (
+		args = ctx.Args()
+		msg  []byte
+		addr string
+	)
+
+	switch {
+	case ctx.IsSet("msg"):
+		msg = []byte(ctx.String("msg"))
+
+	case ctx.Args().Present():
+		msg = []byte(ctx.Args().First())
+		args = args.Tail()
+
+	default:
+		return fmt.Errorf("msg argument missing")
+	}
+
+	switch {
+	case ctx.IsSet("address"):
+		addr = ctx.String("address")
+
+	case ctx.Args().Present():
+		addr = args.First()
+		args = args.Tail()
+
+	default:
+		return fmt.Errorf("address argument missing")
+	}
+
+	resp, err := walletClient.SignMessageWithAddr(ctxc,
+		&walletrpc.SignMessageWithAddrRequest{
+			Msg:  msg,
+			Addr: addr,
+		},
+	)
+	if err != nil {
+		return err
+	}
+
+	printRespJSON(resp)
+
+	return nil
+}
+
+var verifyMessageWithAddrCommand = cli.Command{
+	Name: "verifymessage",
```

Comment:
same here, dup name and missing category field.

Reply:
same as comment above, its part of the `lncli wallet address` cmd suite